app/data_models/questionnaire_store.py

```diff
@@ -48,15 +53,15 @@ def set_metadata(self, to_set):
 
         return self
 
-    def _deserialize(self, data):
+    def _deserialize(self, data: str) -> None:
         json_data = json_loads(data)
         self.progress_store = ProgressStore(json_data.get("PROGRESS"))
         self.set_metadata(json_data.get("METADATA", {}))
         self.answer_store = AnswerStore(json_data.get("ANSWERS"))
         self.list_store = ListStore.deserialize(json_data.get("LISTS"))
         self.response_metadata = json_data.get("RESPONSE_METADATA", {})
 
-    def serialize(self):
+    def serialize(self) -> Any:
```

Comment:
Would be fixed by typing `json_dumps`

Reply:
Cannot be typed cause it uses standard library `dumps`.